packages/compass-home/package.json

```diff
@@ -6,152 +6,66 @@
   "description": "Home",
   "main": "lib/index.js",
   "exports": {
-    "webpack": "./src/index.js",
+    "webpack": "./src/index.ts",
     "require": "./lib/index.js"
   },
   "scripts": {
-    "clean": "rimraf lib",
+    "clean": "rimraf ./lib",
     "precompile": "npm run clean",
-    "compile": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js",
-    "prestart": "electron-rebuild --force --only keytar",
-    "start": "webpack-dev-server --config ./config/webpack.dev.config.js",
-    "start:watch": "npm run clean && webpack --config ./config/webpack.watch.config.js",
-    "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\"",
-    "test:watch": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\" --watch",
-    "test:karma": "xvfb-maybe cross-env NODE_ENV=test karma start",
+    "prewebpack": "rm -rf ./lib",
+    "webpack": "webpack-compass",
+    "node-rebuild": "node ../../scripts/rebuild.js keytar",
```

Comment:
Doesn't seem like there is an elecron-rebuild happening somewhere in other scripts?

Reply:
Removed 👍